extensions/pom.xml

```diff
@@ -98,8 +98,7 @@
         <module>mail</module>
         <module>microprofile-health</module>
         <module>microprofile-metrics</module>
-<!--        Disable MongoDB until quarkus uses mongbd 4.x -->
-<!--        <module>mongodb</module>-->
+        <!--        <module>mongodb</module>-->
```

Comment:
Would you please file a followup issue? I think one is enough for all mongo extensions.

Reply:
https://github.com/apache/camel-quarkus/issues/1064